src/components/NcAppNavigationSettings/NcAppNavigationSettings.vue

```diff
@@ -27,6 +27,9 @@
 		<div id="app-settings__header">
 			<button class="settings-button"
 				type="button"
+				:aria-expanded="open"
+				aria-controls="app-settings__content"
+				:aria-label="t('Open settings menu')"
```

Comment:
```suggestion
				:aria-label="ariaLabel"
```
and below in the computed section add:
```js
ariaLabel() {
	return t('Open settings menu')
},
```
then run `npm run l10n:extract` to extract the string into the translation files.

Reply:
Thank for clarifying, fixed.